### PR TITLE
Added magic-potion question

### PR DESCRIPTION
**Pull request by shannonpaul3**

### CODE REVIEW

File: questions/magic-potion.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Question
+
+Welcome to **Magic Potion**! Here is the problem statement:
+
+Sabrina the witch has a list of magical ingredients that she uses to create potions. Each ingredient has a unique magical power level.
+
+The list of ingredients is sorted in increasing order based on their magical power levels.
+
+One day, Sabrina received an order for a magic potion. The order specified the exact magical power level required. Now, Sabrina needs to choose some ingredients from her list such that the sum of their magical power levels equals the required level.
```
Comment (marceljc3) [inline questions/magic-potion.md:11]: Good story for question context

Comment (dazaiworms) [inline questions/magic-potion.md:11]: the idea is really fun but the explanation of how to use magic power levels and why this aspect is relevant is confusing. i think making the objective would be helpful

File: questions/magic-potion.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+as both combinations sum up to 5.
+
+Here are some criteria for marking someone's code:
+
+- Correctness: Does the code produce correct results for all test cases?
+- Readability: Is the code easy to read and understand?
+- Efficiency: Does the code run efficiently?
+- Style: Does the code follow good coding style and conventions?
+
+Here is a sample solution in Python:
```
Comment (marceljc3) [inline questions/magic-potion.md:52]: looks good, cool question

**Review by marceljc3 — COMMENTED** [review]
a

File: questions/magic-potion.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# Magic Potion
```
Comment (oceansync) [inline questions/magic-potion.md:1]: Nice problem 👍

**Review by oceansync — COMMENTED** [review]
Good documentation of problem, well done 💯 👍

**Pull request merged**